packages/target-decisioning-engine/src/utils.js

```diff
@@ -38,23 +38,12 @@ export function getRuleKey(rule) {
   return rule.ruleKey;
 }
 
-export function parseURL(url) {
-  if (!isString(url)) {
-    // eslint-disable-next-line no-param-reassign
-    url = "";
-  }
-
-  const parsed = parseURI(url) || {};
-
-  const { host = "", path = "", query = "", anchor = "" } = parsed;
-
-  const result = {
-    url,
-    path,
-    query,
-    fragment: anchor
-  };
-
+/**
+ * @param {string} host
+ * @returns {{subdomain?:string, domain?: string, topLevelDomain?: string}}
```

Comment:
change the returns to  `@returns {import("../types/DecisioningContext").DomainContext}` since it's already described in the `DomainContext` type.